contributed_definitions/NXapm_hitmultiplicity_filter.nxdl.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXapm_hitmultiplicity_filter" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
```

Comment:
this base class looks much more generic than an apm_hitmulticiplity_filter. If the type of "hit_multiciplities" would be NX-NUMBER and its name would be match, this definition could be a matching_filter

Reply:
name=MATCH

in appdef:
name=hit_multiplicities type=NXmatching_filter